walk-in-interview/src/main/webapp/log-in/business/script.js

```diff
@@ -55,4 +62,51 @@ backButton.addEventListener('click', (_) => {
 const submitButton = document.getElementById('submit');
 submitButton.addEventListener('click', (_) => {
   // TODO(issue/78): send the account & password input to the firebase auth related stuff
+  const account = document.getElementById('account-input').value;
+
+  if (account === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message',
+        /* msg= */ STRINGS['empty-email'],
+        /* includesDefault= */false);
+    return;
+  }
+
+  const password = document.getElementById('password-input').value;
```

Comment:
do we need to add a check for empty password field?

Reply:
i think no need, since if it is empty, it is regarded as wrong password